manim/mobject/types/vectorized_mobject.py

```diff
@@ -2003,8 +2003,14 @@ def construct(self):
                         (gr-circle_red).animate.shift(RIGHT)
                     )
         """
-        if not all(isinstance(m, (VMobject, OpenGLVMobject)) for m in vmobjects):
-            raise TypeError("All submobjects must be of type VMobject")
+        for m in vmobjects:
+            if not isinstance(m, (VMobject, OpenGLVMobject)):
+                raise TypeError(
+                    f"All submobjects of {self.__class__.__name__} must be of type VMobject. "
+                    f"Got {repr(m)} ({type(m)}) instead. "
```

Comment:
```suggestion
                    f"Got {repr(m)} ({type(m).__name__}) instead. "
```
Maybe slightly better looking?

Reply:
Works for me. `type()` gives the full path, which is more explicit. But then again, we are already using `__name__` for the other type, and I've seen it in other places in the codebase.